utilities/pytest_utils.py

```diff
@@ -230,3 +243,28 @@ def get_cnv_version_explorer_url(pytest_config):
         if not version_explorer_url:
             raise MissingEnvironmentVariableError("Please set CNV_VERSION_EXPLORER_URL environment variable")
         return version_explorer_url
+
+
+def update_storage_class_matrix_config(session, pytest_config_matrix):
+    cmdline_storage_class = session.config.getoption(name="storage_class_matrix")
+    matrix_list = pytest_config_matrix
+    matrix_names = [[*sc][0] for sc in pytest_config_matrix]
+    invald_sc = []
+    if cmdline_storage_class:
+        cmdline_storage_class_matrix = cmdline_storage_class.split(",")
+        if HOSTPATH_CSI in cmdline_storage_class and StorageClassNames.TOPOLVM in cmdline_storage_class:
+            raise ValueError(
+                f"{HOSTPATH_CSI} storage classes can't be used with {StorageClassNames.TOPOLVM} "
+                f": {cmdline_storage_class}"
+            )
+        for sc in cmdline_storage_class_matrix:
+            if sc not in matrix_names:
+                if sc in HPP_STORAGE_CLASSES.keys():
```

Comment:
Could you please add an example of a `--storage-class-matrix` for this condition